examples/vhdl/external_buffer/run.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I'd say that it is already added. Both the `external_buffer` example and the `tb_byte_vector_ptr` are being actually executed in Travis: https://travis-ci.org/VUnit/vunit/jobs/544507378. `external_buffer` is skipped in `py27-acceptance-ghdl` because mcode backend is used, and foreign language is not supported: https://travis-ci.org/VUnit/vunit/jobs/544507377. `supports_vhpi` is used in the `simulator_check`, which was added in 3488.

I did not add it to the 'examples' list of the docs, tho. This is on purpose. I do not want to do it until you tell me that the codebase is ready to be merged, because some descriptions/variables are subject to be modified according to your feedback.